x-pack/plugins/cross_cluster_replication/public/app/components/index.js

```diff
@@ -12,9 +12,10 @@ export { AutoFollowPatternForm } from './auto_follow_pattern_form';
 export { AutoFollowPatternDeleteProvider } from './auto_follow_pattern_delete_provider';
 export { AutoFollowPatternPageTitle } from './auto_follow_pattern_page_title';
 export { AutoFollowPatternIndicesPreview } from './auto_follow_pattern_indices_preview';
-export { FollowerIndexPauseProvider } from './follower_index_pause_provider';
-export { FollowerIndexResumeProvider } from './follower_index_resume_provider';
-export { FollowerIndexUnfollowProvider } from './follower_index_unfollow_provider';
+export { FollowerIndexPauseProvider } from './follower_index_actions_providers/follower_index_pause_provider';
```

Comment:
nit: I haven't looked too closely at the folder structure of CCR, but we typically have an `index.js` file to export files in a directory. In this case, `follower_index_actions_providers` would have an `index.js` file, where you would export `FollowerIndexPauseProvider`, `FollowerIndexResumeProvider`, etc. Then, here, you could export everything from `./follower_index_actions_providers`.

Reply:
thanks for the tip, @alisonelizabeth ! I added `index.js` file and actually removed the individual imports for pause, resume and unfollow providers, since they are not needed anymore.